lib/buildFiles.js

```diff
@@ -25,7 +25,7 @@ var _ = require('lodash'),
  * @returns {null}
  */
 function buildFiles(dictionary, platform) {
-  if (platform.buildPath && platform.buildPath.slice(-1) !== '/') {
+  if (platform.buildPath && (platform.buildPath.slice(-1) !== '/' || platform.buildPath.slice(-1) !== '\\')) {
```

Comment:
I think the logic should be `&&` instead of `||` because the positive case is the one that throws the error (so the logic is a bit backwards, the positive case is the error case).